docs/Esport Protocol V1/Tech integration/rewards.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 4
+---
+
+# Rewards Endpoints
+
+The Rewards Endpoints facilitates the management of user rewards within the platform. This API allows users to retrieve information about their claimable rewards and to claim those rewards as they achieve various milestones.
+
+## API Documentation: Claimable Rewards Information API
+
+### Endpoint: GET `/fe/esports/rewards/claim`
+This endpoint is designed to provide users with information about the claimable rewards they have earned. It's a key feature for platforms that offer reward-based incentives or achievements.
+
+##### Request Header
+- `Content-Type`: application/json
+- `user-jwt-token`: Users JWT token for authentication.
+
+#### Response Structure
+The server responds with a JSON object that includes details about the user's total claimable rewards and specific information about each claimable reward.
+
+- `total_claimable_rewards`: An integer value representing the total amount of rewards that the user can claim.
+- `claimable_rewards_info`: An array of objects (`reward_info`), each containing detailed information about individual claimable rewards.
+
+##### Response Status Code
+- `Success`: 200 OK
+- `Failure`: 
+  - `401 Unauthorized`
+    - "UN_AUTHORIZED"
+
+##### Example Response
+```json
+{
+  "total_claimable_rewards": 100,
+  "claimable_rewards_info": [
+    {
+      
+      "user_id":"test1",
+      "tournament_id":"testTournament5",
+      "claim_status":"CLAIMED_SUCCESSFULLY",
```

Comment:
@Jitendra-08 And here, the API purpose is to get the claimable rewards of the users, so ideally reward state should be **UNCLAIMED**